lib/media/streaming_engine.js

```diff
@@ -237,6 +237,7 @@ shaka.media.StreamingEngine.PlayerInterface;
  *   updateTimer: ?number,
  *   waitingToClearBuffer: boolean,
  *   waitingToFlushBuffer: boolean,
+ *   deferredClearBufferOffset: number,
```

Comment:
I think a better name would be `clearBufferSafeMargin`, or something including safeMargin, to be consistent.